greenplumpython/table.py

```diff
@@ -52,6 +52,26 @@ def select(self, target_list: Iterable) -> "Table":
             parents=[self],
         )
 
+    def join(
+        self,
+        other: "Table",
+        expr: expr.Expr,
+        how: Optional[str] = "natural",
+        target_list: Optional[Iterable] = None,
+    ) -> "Table":
+        select_str = (
+            ",".join([str(target) for target in target_list]) if target_list is not None else "*"
+        )
+        on_str = "ON " + str(expr) if how.upper() != "NATURAL" else None
```

Comment:
this `on_str` can be `None`
it this fit for
```
                SELECT {select_str} 
                FROM {self.name} {how.upper()} JOIN {other.name}
                {str(on_str)}  
```
str(None) == 'None'

Reply:
Because when we have `Natural Join` then `ON` statement is not required